src/utils/formatNumber.ts

```diff
@@ -1,10 +1,12 @@
-export function formatNumber(value: number, maxDigits?: number): string {
+export function formatNumber(value: number, maxDigits = 3, isHashrate = true): string {
     //TODO: add props for customisation
-    return Intl.NumberFormat(undefined, {
+    const formatted = Intl.NumberFormat(undefined, {
         notation: 'compact',
-        maximumFractionDigits: maxDigits || 3,
+        maximumFractionDigits: maxDigits,
         style: 'decimal',
     }).format(value);
+    // workaround because there is no "hashrate" unit
+    return isHashrate ? formatted.replace('B', 'G') : formatted;
```

Comment:
This will only work for english locale. B is different in other locales

Reply:
You're right. I'll find better solution to support locales and hashrate